Scripts/update-cocoa.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Maybe these three scripts into one, with config being the input script, and all of the `sourcing` the common script. Removes duplication a bit. See for example https://github.com/getsentry/sentry-react-native/blob/main/scripts/update-wizard.sh that's the "updater.yml" target script and itself includes a common `update-package-json.sh`

Reply:
I've raised a separate issue for this